### PR TITLE
Blog home design

### PR DESCRIPTION
**Pull request by danutsss**
**Changes:**

- changed font from `Roboto` to `Montserrat`;
- now, every post has its own row;

### CODE REVIEW

File: src/app/components/home/post-detail/post-detail.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.word-break {
+	word-break: break-all !important;
```
Comment (MirceaDumitrescu) [inline src/app/components/home/post-detail/post-detail.component.scss]: Why do you need the !important marker?

File: src/app/components/home/post-list/post-list.component.html [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+            <div class="text-center mt-4 mb-4">
+                <h1 class="fw-bolder">Ultimele postari</h1>
+            </div>
             <div *ngIf="posts$ | async as posts" class="row">
                 <div class="row" *ngIf="posts.length > 0">
-                    <div *ngFor="let post of posts" class="col-lg-6">
+                    <div *ngFor="let post of posts">
                         <mat-card class="mb-4">
-                            <mat-card-title>{{ (post.title.length > 30) ? (post.title | slice:0:30) + ' ...' :
+                            <mat-card-title>{{ (post.title.length > 50) ? (post.title | slice:0:50) + '...' :
```
Comment (MirceaDumitrescu) [inline src/app/components/home/post-list/post-list.component.html]: You could create a PIPE in Angular for this type of formatting

File: src/app/components/home/post-detail/post-detail.component.scss (hunk 1 above)
Comment (danutsss) [reply src/app/components/home/post-detail/post-detail.component.scss]: It was a typo, I've copy-pasted the rule from another file and forgot to remove that marker, thank you!

File: src/app/components/home/post-list/post-list.component.html (hunk 2 above)
Comment (danutsss) [reply src/app/components/home/post-list/post-list.component.html]: @MirceaDumitrescu thank you for your idea. I've created a custom pipe.

**Review by MirceaDumitrescu — APPROVED** [review]

**Pull request merged**